### PR TITLE
Sin ,cos, relu, gelu and logical_or_ new infrastructure sweeps

### PR DESCRIPTION
**Pull request by npetrovic-tenstorrent**
Add new Python sweeps for ops:
- relu
- gelu
- logical_or_
- cos
- sin

### CODE REVIEW

File: .github/workflows/ttnn-run-sweeps.yaml [hunk 1]
```diff
@@ -13,6 +13,10 @@ on:
           - add
           - line_all_gather
           - logical_and_
+          - eltwise.logical_or_
+          - eltwise.relu
+          - eltwise.gelu
+          - eltwise.cos
```
Comment (jdesousa-TT) [inline .github/workflows/ttnn-run-sweeps.yaml:19]: eltwise.sin is missing from this list

**Review by jdesousa-TT — CHANGES_REQUESTED** [review]

**Comment by jdesousa-TT** [comment]
Running the new sweeps here:
logical_or https://github.com/tenstorrent/tt-metal/actions/runs/10906953084
relu https://github.com/tenstorrent/tt-metal/actions/runs/10906956537
gelu https://github.com/tenstorrent/tt-metal/actions/runs/10906958671
cos https://github.com/tenstorrent/tt-metal/actions/runs/10906960913

File: .github/workflows/ttnn-run-sweeps.yaml (hunk 1 above)
Comment (npetrovic-tenstorrent) [reply .github/workflows/ttnn-run-sweeps.yaml:19]: Thanks! I will correct this!

Comment (npetrovic-tenstorrent) [reply .github/workflows/ttnn-run-sweeps.yaml:19]: Corrected in the latest commit

**Review by jdesousa-TT — APPROVED** [review]

**Pull request merged**